### PR TITLE
Document homepage

### PR DESCRIPTION
**Pull request by flowerncsu**

### CODE REVIEW

**Comment by mlavin** [comment]
Test failure looks unrelated to this change (and non-deterministic ¯\(°_o)/¯ ). Needs to be addressed eventually but not necessarily here.

File: docs/homepage.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+will appear in that place.
+
+When referencing the content or description of the entry, you will want to use `content_rendered` or 
+`description_rendered`, respectively, as those have been parsed to be html-ready. You will also want to use the Django
+template function `safe` to ensure that Django does not duplicate the rendering that has already been done. The end
+result will look something like `{{ entry.entry.content_rendered | safe }}`
+
+There are two options for using the resulting context data in the template: iterating or retrieving individual entries.
+
+#### Iterating
```
Comment (mlavin) [inline docs/homepage.md]: It's probably worth noting the `collection_limit` on the view defines how many will be available in the template.

Comment (flowerncsu) [inline docs/homepage.md]: That doesn't actually affect the `featured_entry_set` that I'm referencing here. `collection_limit` only applies to the context variable indicated by `context_label`

Comment (mlavin) [inline docs/homepage.md]: Yes you are right. Then that should be noted. Anything published in the collection will be shown which is different behavior from the  base `ConfiguredCollectionMixin`.

File: docs/homepage.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+class MyNewPage(FeaturedEntryMixin, TemplateView):
+    template_name = "website/my_template.html"
+    featured_config_key = 'MY_NEW_COLLECTION'
+```
+
+That will point the `FeaturedEntryMixin` to the correct collection, and include the entries in the context.
+
+### Use in Template
+
+The resulting entries will be available in the template as `featured_entry_set`.
```
Comment (mlavin) [inline docs/homepage.md]: There is also a reference to the first item in this collection via `featured_entry`.

**Comment by mlavin** [comment]
:sunflower:

**Pull request merged**